Makefile

```diff
@@ -3,7 +3,7 @@ SERVER=""
 
 install:
 	python3.6 -m ensurepip
-	pkg install -q -y libgit2
+	pkg install -q -y libgit2 libucl cython3
```

Comment:
this works, but only if we don't use our own venv…
which, per our docs, we don't (recommend)